### PR TITLE
removed EDS egress references

### PR DESCRIPTION
**Pull request by rmuolic**

### CODE REVIEW

**Comment by rmuolic** [comment]
@osisoft-mbishop and @aveva-jhwang , please review the topics that have been updated to remove the EDS egress content/references. Thanks.

File: shared-content/configuration/egress-endpoints.md [hunk 1]
```diff
@@ -8,15 +8,12 @@ PI adapters collect time series data, which  they can send to a permanent data s
 
 - OSIsoft Cloud Services (OCS)
 - PI servers through PI Web API
-- Edge Data Store (EDS)
```
Comment (aveva-jhwang) [inline shared-content/configuration/egress-endpoints.md:11]: Could we include a note saying that we do plan on supporting egress to Edge Data Store in the future? I don't know how to best phrase it.

Comment (rmuolic) [reply shared-content/configuration/egress-endpoints.md:11]: @aveva-jhwang , something like the following?

```suggestion
Edge Data Store (EDS)*

* PI adapters will support egress to Edge Data Store (EDS) in the future.
```

**Review by osisoft-mbishop — APPROVED** [review]
Changes look good Rainier, nice job.

File: shared-content/configuration/egress-endpoints.md [hunk 2]
```diff
@@ -64,16 +61,14 @@ The following parameters are available for configuring egress endpoints:
...
 | **ClientId**                    | Required for OCS endpoint | `string`    | Authentication with the OCS OMF endpoint <br><br>Allowed value: any string, can be null if the endpoint URL schema is `HTTP`<br>Default: `null`|
 | **ClientSecret**                | Required for OCS endpoint | `string`    | Authentication with the OCS OMF endpoint <br><br>Allowed value: any string, can be null if the endpoint URL schema is `HTTP`<br>Default: `null`|
 | **TokenEndpoint**               | Optional for OCS endpoint | `string`    | Retrieves an OCS token from an alternative endpoint <br><br>Allowed value: well-formed http or https endpoint string <br>Default value: `null` |
 | **ValidateEndpointCertificate** | Optional                  | `boolean`   | Disables verification of destination certificate. **Note:** Only use for testing with self-signed certificates. <br><br>Allowed value: `true` or `false`<br>Default value: `true` |
 
-**Note:** If the URL schema for the egress endpoint is HTTP, which is typical for EDS, then you do not need to specify credentials.
```
Comment (aveva-jhwang) [inline shared-content/configuration/egress-endpoints.md:75]: For this line, let's just remove the "which is typical for EDS" rather than the entire line. I'm not familiar with the technical details around OCS endpoint but HTTP could apply outside of just EDS.

File: shared-content/troubleshooting/troubleshooting.md [hunk 3]
```diff
@@ -28,17 +28,6 @@ Perform the following steps to view the System and OmfEgress logs:
 
 2. Optional: Change the log level of the adapter to receive more information and context. For more information, see [Logging configuration](xref:LoggingConfiguration).
 
-### ASP .NET Core platform
```
Comment (aveva-jhwang) [inline shared-content/troubleshooting/troubleshooting.md:31]: For this section, I need to double check whether this is only applicable with EDS or other endpoints. It says "in case EDS encounters heartbeat failures **OR** if the adapter throughput is too high." We might just have to remove the EDS heartbeat failures portion.

File: shared-content/configuration/egress-endpoints.md (hunk 1 above)
Comment (aveva-jhwang) [reply shared-content/configuration/egress-endpoints.md:11]: Maybe "While this version of the PI Adapter does not support egress to Edge Data Store (EDS), it will be supported for the next version of EDS."?

File: shared-content/configuration/egress-endpoints.md (hunk 2 above)
Comment (osisoft-mbishop) [reply shared-content/configuration/egress-endpoints.md:75]: @aveva-jhwang, we should get a dev review on this. Using HTTP with any endpoint other than EDS is likely a security vulberability.

Comment (aveva-jhwang) [reply shared-content/configuration/egress-endpoints.md:75]: Good point. @Xiaoli-OSI @zryska - could you comment on this particular statement?

File: shared-content/configuration/egress-endpoints.md (hunk 1 above)
Comment (osisoft-mbishop) [reply shared-content/configuration/egress-endpoints.md:11]: @aveva-jhwang and @rmuolic, I think this statement would be better in a release note or a known issue rather than baking it into the docs. Do you agree?

Comment (rmuolic) [reply shared-content/configuration/egress-endpoints.md:11]: @osisoft-mbishop I agree with that as well. It doesn’t seem like a statement to put into a user guide.

**Comment by rmuolic** [comment]
Added @Xiaoli-OSI and @zryska for their review and input as well. FYI @aveva-jhwang and @osisoft-mbishop

File: shared-content/configuration/egress-endpoints.md (hunk 2 above)
Comment (zryska) [reply shared-content/configuration/egress-endpoints.md:75]: I am okay either way - This has been added mainly to discourage users to use real credentials when configuring EDS - to not have it exposed in plain-text due to nature of HTTP protocol. Currently we don't have any other HTTP OMF endpoint available  (OCS and PI Web API are both HTTPS only)  so removing it entire sentence would work. (we might want to phrase it differently to stress the security aspect of it in the future when EDS is updated)

Comment (rmuolic) [reply shared-content/configuration/egress-endpoints.md:75]: Thanks @zryska , @aveva-jhwang I'm okay with removing the sentence if you agree.

File: shared-content/troubleshooting/troubleshooting.md (hunk 3 above)
Comment (rmuolic) [reply shared-content/troubleshooting/troubleshooting.md:31]: @zryska for this section, should we remove just the "in case EDS encounters heartbeat failures" or should we remove the section entirely?

The ASP .NET Core platform log provides information from the Kestrel web server that hosts the application. The log will only be present in case EDS encounters heartbeat failures or if the adapter throughput is too high. Perform the following steps to spread the load among multiple adapters:

1. Decrease the scan frequency.
2. Lower the amount of data selection items.

File: shared-content/configuration/egress-endpoints.md [hunk 4]
```diff
@@ -64,16 +61,14 @@ The following parameters are available for configuring egress endpoints:
...
-| **Username**                    | Required for PI server and EDS endpoint  | `string`    | Basic authentication to the PI Web API OMF or EDS endpoint <br><br>_PI server:_<br>Allowed value: any string<br>Default: `null`<br>**Note:** If your username contains a backslash, you must add an escape character, for example, type `OilCompany\TestUser` as `OilCompany\\TestUser`.<br><br>_EDS:_<br>Allowed value: any string, can be null if the endpoint URL schema is `HTTP` |
-| **Password**                    | Required for PI server and EDS endpoint  | `string`    | Basic authentication to the PI Web API OMF or EDS endpoint <br><br>_PI server:_<br>Allowed value: any string<br>Default: `null`<br><br>_EDS:_<br>Allowed value: any string, can be null if the endpoint URL schema is `HTTP`  |
+| **Endpoint**                    | Required                  | `string`    | Destination that accepts OMF v1.2 messages. Supported destinations include OCS, and PI Server.<br><br>Allowed value: well-formed http or https endpoint string<br>Default: `null` |
```
Comment (Xiaoli-OSI) [inline shared-content/configuration/egress-endpoints.md:64]: the "," before "and PI Server" can be removed.

File: shared-content/configuration/egress-endpoints.md (hunk 2 above)
Comment (zryska) [reply shared-content/configuration/egress-endpoints.md:75]: Sounds good to me.

File: shared-content/troubleshooting/troubleshooting.md (hunk 3 above)
Comment (zryska) [reply shared-content/troubleshooting/troubleshooting.md:31]: This is still relevant for EDS - but it's probably confusing to have it in the adapter documentation so I would agree with removing it from here since it should be mentioned only in EDS user guide.

Comment (rmuolic) [reply shared-content/troubleshooting/troubleshooting.md:31]: @zryska thank you but to confirm, we should remove the entire section or only "in case EDS encounters heartbeat failures"?

File: shared-content/configuration/egress-endpoints.md (hunk 4 above)
Comment (rmuolic) [reply shared-content/configuration/egress-endpoints.md:64]: @Xiaoli-OSI , to confirm your changes, this sentence should be: "Supported destinations include OCS."?

File: shared-content/troubleshooting/troubleshooting.md (hunk 3 above)
Comment (zryska) [reply shared-content/troubleshooting/troubleshooting.md:31]: We might rephrase it to something like this:

The ASP .NET Core platform log provides information from the Kestrel web server that hosts the application. The log could contain information that the adapter is getting overloaded with incoming data. Perform the following steps to spread the load among multiple adapters:

1. Decrease the scan frequency.
2. Lower the amount of data selection items.

File: shared-content/configuration/egress-endpoints.md (hunk 1 above)
Comment (rmuolic) [reply shared-content/configuration/egress-endpoints.md:11]: Resolved. OK to remove the Edge Data Store (EDS) bullet point from the list.

File: shared-content/configuration/egress-endpoints.md (hunk 4 above)
Comment (Xiaoli-OSI) [reply shared-content/configuration/egress-endpoints.md:64]: I meant remove the comma.
